modules/lss-creditserweiterung/creditserweiterung.user.js

```diff
@@ -0,0 +1,159 @@
+            updateMessage: 'Waarden actualiseren<br>om de 5 minuten.',
+            allianceFunds: 'Teamkas'
+        }
+    };
+
+    function getCreditsOfNextRank(earnedCredits){
+        let ranks = I18n.t('lssm.creditserweiterung.ranks');
+        for (var key in ranks) {
+            if(earnedCredits > key){
+                continue;
+            } else {
+                return key;
+            }
+        }
+    }
+
+    function getEarnedCredits() {
+        let response = $.ajax({
+            type: "GET",
+            url: $('#navbar_profile_link').attr('href'),
+            async: false
+        }).responseText;
+        let results = $(response).find('.page-header').text().trim().match(I18n.t('lssm.creditserweiterung.earnedCreditsRegEx'));
+        return parseInt(results[1].replace(/\./g,'').replace(/,/g, ''));
+    }
+
+    function getAlianceFundsCredits() {
+        let response = $.ajax({
+            type: "GET",
+            url: "./verband/kasse",
+            async: false
+        }).responseText;
+        let results = $(response).find('h1');
+        if (results[0]){
+            let result = results[0].innerText.replace(/\./g, "").replace(/,/g, "");
+            return parseInt(result);
+        }
+        return
+    }
+
+    function updateCredits() {
+        $("#navigation_top").show();
+        $("#coins_top").show();
+        let currentCredits = $("#navigation_top").clone().wrap('<li>').parent().html().replace("navigation_top", "creditserweiterung_credits");
+        let currentCoins = $("#coins_top").clone().wrap('<li>').parent().html().replace("coins_top", "creditserweiterung_coins");
+        let earnedCredits = getEarnedCredits();
+        let creditsOfNextRank = getCreditsOfNextRank(earnedCredits);
+        let nextRank = I18n.t('lssm.creditserweiterung.ranks')[creditsOfNextRank];
+        var creditsToNextRank = creditsOfNextRank - earnedCredits;
+        let allianceFundsCredits = getAlianceFundsCredits();
+
+        let markup = '<li><a id="menu_creditsverwaltung"  class="dropdown_toggle href="#" role="button" data-toggle="dropdown" aria-expanded="false">';
+        markup += '<img id="ls-credits-money-img" style="height: 19px; width: 19px; cursor: pointer;" src="';
+        markup += lssm.getlink("/modules/lss-creditserweiterung/img/icons8-money-box-150.png") + '">';
+        markup += '<span class="visible-xs">' + I18n.t('lssm.creditserweiterung.texts.dropdownName') + '</span>';
+        markup += '<b class="caret"></b></a><ul class="dropdown-menu" role="menu" aria-labelledy="menu_Creditsverwaltung>';
+        markup += '<li class="divider" role="presentation"></li><li role="presentation" >' + currentCredits + '</li>';
+        markup += '<li role="presentation"><a href="/credits/overview" class="lightbox-open" target="blank">' + I18n.t('lssm.creditserweiterung.texts.creditsOverview') + '</a></li>';
+        markup += '<li role="presentation">' + currentCoins + '</li>';
+
+        if (true){
```

Comment:
if(true) ist immer wahr ;-)
Ich hatte das nur in meinem Beispiel eingebaut, weil ich deine ursprüngliche IF/ELSE Bedingung nicht verstanden habe und dir einen Weg aufzeigen wollte, um Code Redundanzen zu sparen.

Reply:
Das ist jetzt egal, denn ic hab festgestellt, dass es das Coinsprotokoll auch in den anderen Versionen gibt, auch wenn ichs bisher nie gefunden hab :D